docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -415,6 +414,194 @@ children:
     url: Download-examples.html
     output: 'web, pdf'
 
+- title: 'Connectors reference'
+  url: Connectors-reference.html
```

Comment:
https://loopback.io/doc/en/lb4/Connectors-reference.html does not exist. Am I missing something here?

Reply:
Just realized https://github.com/strongloop/loopback.io/pull/863

Reply:
There are many more "external" files, all of which are present in the `lb3` dir or made available there eventually. Those files are copied over to the `lb4` dir by the build process in `loopback.io`.